pom.xml

```diff
@@ -41,6 +50,27 @@
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>
+                    <compilerArgument>-Xlint:unchecked</compilerArgument>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
What is the advantage of the Maven assembly plugin over the Maven shade plugin?

Reply:
The maven assembly plugin allows the compilation of a fat jar which includes the exploded inclusion of all dependencies in the distributable artifact.  This allows us to easily install it in the plugin directory of the server.

Does the maven shade plugin do something similarly?  What are the advantages of the shade plugin?

Reply:
The shade plugin is a more robust compared to the assembly plugin (see examples [here](http://maven.apache.org/plugins/maven-shade-plugin/) and [here](http://maven.apache.org/plugins/maven-assembly-plugin/examples/index.html)) though it probably doesn't make a difference here.

Reply:
Oh!  Good to know!  I'll read the documents at those links.  Do they include any examples/tutorials on usage?